src/js/claims-status/components/appeals-v2/Docket.jsx

```diff
@@ -1,10 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class Docket extends React.Component {
-  render() {
-    return (null);
-  }
+function Docket({ ahead, total }) {
```

Comment:
A bit paranoid probably, but what happens if we're missing either of these? We may want to just not show the component / have it return `null` or something since it wouldn't make much sense to show anything here with a number missing. 🤷‍♂️

Reply:
Good call. We currently don't have any logic surrounding _when_ to show the docket, but when we get that, I'm guessing it won't be a problem (probably that logic will say something like `if the current status is one of [these], and there's information in the docket object, render <Docket/>`).

I can make a comment for paranoia's sake for sure, and if our logic surrounding when to render the document doesn't involve checking `appeal.attributes.docket`, then we can do that check in here.